source/_patterns/10-components/video/video.scss

```diff
@@ -123,11 +123,25 @@
 
 .article-video__player {
   background-color: #000;
+
+  iframe {
```

Comment:
war glaubs schon vorher drin.. sollten wir da nicht dont touch, is ja vom SRG Player generiert?